src/trace-api.ts

```diff
@@ -241,7 +242,7 @@ export class StackdriverTracer implements Tracer {
       rootContext = UNTRACED_ROOT_SPAN;
     } else {
       // Create a new root span, and invoke fn with it.
-      rootContext = new RootSpanData(
+      const rootSpan = new RootSpanData(
```

Comment:
Why did you change this and add a new variable? Assigning to `rootContext` here should still work right?

Reply:
No, since `rootContext` is not a `RootSpanData`, and won't have a known `assignTraceWriter` field.